tests/unit/CMakeLists.txt

```diff
@@ -8,7 +9,7 @@ endif()
 ## Base tests
 foreach(test_name tests_high_five_base tests_high_five_multi_dims tests_high_five_easy test_all_types)
   add_executable(${test_name} "${test_name}.cpp")
-  target_link_libraries(${test_name} HighFive Catch2::Catch2WithMain)
+  target_link_libraries(${test_name} HighFive HighFiveWarnings Catch2::Catch2WithMain)
```

Comment:
Maybe we should use the alias syntax: `HighFive::Warnings`, `HighFive::HighFive`, ...

Reply:
That would look neat. We can look into that for `v3.0.0`.